crates/server/src/admin/service.rs

```diff
@@ -69,11 +70,6 @@ impl AdminConfig {
     }
 }
 
-// Embed the contents of the admin-ui build directory into the binary
-#[derive(RustEmbed)]
-#[folder = "../../node-ui/build/"]
-struct NodeUiStaticFiles;
```

Comment:
we don't want to remove this, just that the path is now from the env variable, nothing else needs to change

Reply:
Any idea on how to use it? using env in my case was throwing error that it needs to have a "static path"

Reply:
according to https://docs.rs/rust-embed-impl/8.7.0/src/rust_embed_impl/lib.rs.html#365-368, it calls out to `shellexpand` when the `interpolate-folder-path` feature flag is activated (which it isn't by default)

so this should work (at least in theory):

```suggestion
// Embed the contents of the admin-ui build directory into the binary
#[derive(RustEmbed)]
#[folder = "$CALIMERO_WEB_UI_PATH"]
struct NodeUiStaticFiles;
```

Reply:
Not sure what the issue was, but it works on my end 🤷🏽